### ISSUE TITLE
Suggestion: "size" field for centerTowns

### ISSUE THREAD

**Issue by goldencow2** [18xx-maker/18xx-maker#380]
In order to have the nice "white outline" effect that larger cities have.
![Untitled](https://user-images.githubusercontent.com/43799724/93812558-6f4d3680-fc17-11ea-9c6e-4317c5dee9fb.png)

**Comment by th3tick** [comment]
centerTowns have a white outline.  do you mean a way to make a double centerTown?  You can just do two centerTowns

**Comment by th3tick** [comment]
![image](https://user-images.githubusercontent.com/48138042/93956990-8e7bbf00-fd10-11ea-84b2-22d3b9d10218.png)
    "mine0": {
      "color": "yellow",
      "centerTowns": [
        { "angle": 120, "percent": 0.17 },
        { "angle": -60, "percent": 0.17 }
      ],
      "track": [
        { "side": 1, "type": "straight" },
        { "side": 2, "type": "straight" }
      ]
    },

**Comment by goldencow2** [comment]
Something closer to this aesthetic, though now I realize I could probably just put some centerTowns on top of a two-size city and it would likely look good. Leaving up suggestion as Kelsin told me to put it here in the first place.

![Untitled](https://user-images.githubusercontent.com/43799724/94071563-bb37e100-fdb9-11ea-8848-92650511a6c2.jpg)

**Comment by goldencow2** [comment]
Proof of concept:
![Untitled](https://user-images.githubusercontent.com/43799724/94072198-d820e400-fdba-11ea-8e7c-1363a8a942a2.jpg)
```
      "OE01": {
          "color": "yellow",
          "cities": [
              {
                  "size": 2,
                  "rotate": 90,
                  "width": 15
              }
          ],
          "centerTowns" : [
              {
                  "angle": 180,
                  "percent": 0.2
              },
              {
                  "percent": 0.2
              }
          ],
          "values": [
              {
                  "value": 10,
                  "angle": 90,
                  "percent": 0.6
              },
              {
                  "value": 10,
                  "angle": 270,
                  "percent": 0.6
              }
          ],
          "track": [
              {
                  "side": 1,
                  "type": "straight"
              }
          ]
      }
```

**Comment by th3tick** [comment]
Yeah, that has more lines than cities draw.  By making the color something you can specify:
      "cities": [
        {
          "size": 2,
          "color": "black",
          "outlineColor": "white",
          "width": 15,
          "rotation": 30
        }
      ],
<img width="156" alt="Screen Shot 2020-09-23 at 4 15 29 PM" src="https://user-images.githubusercontent.com/48138042/94075206-02bd6d80-fdb8-11ea-9fcf-0ed192ee09be.png">

This can be done, and I have code that does it.

**Comment by th3tick** [comment]
pull request which generates this waiting

<img width="164" alt="Screen Shot 2020-09-24 at 9 26 35 AM" src="https://user-images.githubusercontent.com/48138042/94166021-14e6ec80-fe48-11ea-97c6-554b38e5dcb0.png">
`    "cT2": {
      "color": "yellow",
      "centerTowns": [
        { "size": 2, "rotation": 30 }
      ],
      "track": [
        { "side": 1, "type": "straight" },
        { "side": 2, "type": "straight" }
      ]
    },
`
